File: src/main/java/com/demonwav/mcdev/platform/forge/sideonly/Side.java
package com.demonwav.mcdev.platform.forge.sideonly;

import org.jetbrains.annotations.NotNull;

public enum Side {

    CLIENT("SideOnly.CLIENT"),
    SERVER("SideOnly.SERVER"),
    NONE("NONE"),
    INVALID("INVALID");

    @NotNull
    private final String name;

    Side(@NotNull String name) {
        this.name = name;
    }

    @NotNull
    public String getName() {
        return name;
    }
}

File: src/main/kotlin/platform/forge/inspections/sideonly/Side.kt
/*
 * Minecraft Dev for IntelliJ
 *
 * https://minecraftdev.org
 *
 * Copyright (c) 2022 minecraft-dev
 *
 * MIT License
 */

package com.demonwav.mcdev.platform.forge.inspections.sideonly

import com.intellij.openapi.util.Key

enum class Side {

    CLIENT,
    SERVER,
    NONE,
    INVALID;

    companion object {
        val KEY = Key<Side>("MC_DEV_SIDE")
    }
}
